### PR TITLE
Add epcsearch.py to packaging practices

### PR DESCRIPTION
**Pull request by JacekJagosz**
It should reduce number of questions at #Solus-dev, and speed up packaging for newbies.

## Description

<Info on what this pull request adds>

### Submitter Checklist

- [ ] Updated the "lastmod" portion at the top of any modified Markdown files.
- [ ] Squashed commits with `git rebase -i` (if needed)

### CODE REVIEW

File: packaging/packaging-practices/en.md [hunk 1]
```diff
@@ -196,6 +196,7 @@ In output:
 Provides: pkgconfig(gtk+-3.0) pkgconfig(gdk-3.0) pkgconfig(gdk-wayland-3.0) pkgconfig(gail-3.0) pkgconfig(gdk-x11-3.0) pkgconfig(gtk+-unix-print-3.0)
 pkgconfig(gtk+-wayland-3.0) pkgconfig(gtk+-x11-3.0) 
 ```
```
Comment (JoshStrobl) [inline packaging/packaging-practices/en.md]: Should be a newline between the paragraph and end of the code block.

Comment (JacekJagosz) [reply packaging/packaging-practices/en.md]: I hope this is what you meant

Comment (JoshStrobl) [reply packaging/packaging-practices/en.md]: No the newline should be between the paragraph and the **end** of the code block (so the ```)

Comment (JacekJagosz) [reply packaging/packaging-practices/en.md]: Sorry, I don't think I get it. Now is that it?

Comment (JoshStrobl) [reply packaging/packaging-practices/en.md]: I meant between the pkgconfigs code block and the paragraph, but I'd keep the other newline you just added too :D

**Comment by JoshStrobl** [comment]
Yay we got there in the end. Thanks for the PR and tolerating my pedantic ways :P

File: packaging/packaging-practices/en.md (hunk 1 above)
Comment (JacekJagosz) [reply packaging/packaging-practices/en.md]: Oh, sorry, I thought about different code block

**Comment by JacekJagosz** [comment]
Great, thank you!

**Pull request merged**